pkg/daemon/ceph/client/crush_rule.go

```diff
@@ -62,6 +76,30 @@ func buildTwoStepPlainCrushRule(crushMap CrushMap, ruleName string, pool cephv1.
 	)
 }
 
+func buildTwoStepHybridCrushRule(crushMap CrushMap, ruleName string, pool cephv1.PoolSpec) string {
+	var primaryOSDDeviceClass string
+	var secondaryOSDsDeviceClass string
+	if pool.Replicated.HybridStorage.PrimaryDeviceClass != "" {
+		primaryOSDDeviceClass = fmt.Sprintf("class %s", pool.Replicated.HybridStorage.PrimaryDeviceClass)
```

Comment:
How about including `class` in the template `twoStepHybridCRUSHRuleTemplate` above?
```suggestion
		primaryOSDDeviceClass = fmt.Sprintf("%s", pool.Replicated.HybridStorage.PrimaryDeviceClass)
```

Reply:
moved `class` in the template.